t4_devkit/schema/tables/object_ann.py

```diff
@@ -82,7 +86,24 @@ class ObjectAnn(SchemaBase):
         converter=lambda x: RLEMask(**x) if isinstance(x, dict) else x,
         validator=validators.optional(validators.instance_of(RLEMask)),
     )
+    orientation: float | None = field(
+        default=None, validator=validators.optional(validators.instance_of(float))
+    )
+    number: int | None = field(
+        default=None, validator=validators.optional(validators.instance_of(int))
+    )
     automatic_annotation: bool = field(default=False, validator=validators.instance_of(bool))
+    autolabel_metadata: list[AutolabelModel] | None = field(
+        default=None,
+        converter=lambda x: [
+            AutolabelModel(**model) if isinstance(model, dict) else model for model in x
+        ]
+        if isinstance(x, list)
+        else x,
+        validator=validators.optional(
+            validators.deep_iterable(validators.instance_of(AutolabelModel))
+        ),
+    )
```

Comment:
[WANT] I want to add post-validation to check if `automatic_annotatoin` represents exactly existence of `autolabel_metadata` in `__attrs_post_init__`, for example:

```python
def __attrs_post_init__(self) -> None:
    # if automatic_annotation=True, autolabel_metadata must be exist
    if self.automatic_annotation and self.autolabel_metadata is None:
        raise TypeError("...")
    # if automatic_annotation=False, autolabel_metadata must not exist
    if self.automatic_annotation and self.autolabel_metadata is not None:
        raise TypeError("...")
```